### ISSUE TITLE
Are there functions similar to torch.nn.L1Loss(), torch.nn.MSELoss() and torch.nn.BCELoss() in oneflow

### ISSUE THREAD

**Issue by 15168387900** [Oneflow-Inc/oneflow#3791]
Are there functions similar to torch.nn.L1Loss(), torch.nn.MSELoss() and torch.nn.BCELoss() in oneflow

**Comment by MARD1NO** [comment]
Thanks for your feedback! We have received your need of these operators! we will develop them as soon as possible. 

In addition, you can also try to use some operators in `oneflow.math` module to build the loss function. 

We will reply to you again when the developing work is all done~
